src/aiida/cmdline/commands/cmd_presto.py

```diff
@@ -94,6 +132,40 @@ def verdi_presto(ctx, profile_name, email):
     else:
         echo.echo_report('RabbitMQ server detected: configuring the profile with a broker.')
 
+    if use_postgres:
+        echo.echo_report('Using PostgreSQL: attempting to connect and create a user and database.')
```

Comment:
might be nice to move all this to a separate helper function, since this command is already quite long and difficult to read. Feel free to ignore.